src/Detector/CineonDetector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Selective\ImageType\Detector;
+
+use Selective\ImageType\ImageType;
+use SplFileObject;
+
+/**
+ * Detector.
+ */
+final class CineonDetector implements DetectorInterface
+{
+    /**
+     * DNG identification.
+     *
+     * @param SplFileObject $file The image file
+     *
+     * @return ImageType|null The image type
+     */
+    public function detect(SplFileObject $file): ?ImageType
+    {
+        $file->rewind();
+        $bytes = bin2hex($file->fread(4) ?: "");
+
+        return $bytes === "802a5fd7" || $bytes === "d75f2a80" ? new ImageType(ImageType::CIN) : null;
```

Comment:
According to the `Cineon` image detection reference, the files begin with signature bytes `80 2A 5F D7`, or `D7 5F 2A 80`.

That is, the first `4` bytes will be `802a5fd7` or `802a5fd7` hex string.